agent/requirements.txt

```diff
@@ -1,6 +1,7 @@
 bottle
 cffi
 click>=8.0
+hiredis
```

Comment:
OK ... why *do* we need this package we apparently don't reference directly? I see that it's some sort of Redis wrapper or extension, but with no references elsewhere ... what *is* going on here?

Reply:
Ugh: https://github.com/redis/redis-py/commit/16cfcc7fced84d2b53edf95af1c40b230b30fc3d

Reply:
I'll file a separate PR for this fix.

Reply:
See PR #2547.